src/webAppCreator.ts

```diff
@@ -11,16 +13,19 @@ import WebSiteManagementClient = require('azure-arm-website');
 import * as WebSiteModels from '../node_modules/azure-arm-website/lib/models';
 import * as util from './util';
 
-export class WebAppCreator extends WizardBase {
-    constructor(output: vscode.OutputChannel, readonly azureAccount: AzureAccountWrapper, subscription: SubscriptionModels.Subscription, persistence?: vscode.Memento) {
+export type WebsiteOS = "linux" | "windows";
+export type AppKind = "app" | "functionapp";
```

Comment:
nit: Did you pick these names or are you using values from Azure?

If you picked them, I would prefer 'webapp' over 'app'. If you didn't pick them - shame on Azure :P

Reply:
Azure values, and of course they've evolved and can't be changed, so I don't blame them.